src/App.js

```diff
@@ -6,6 +6,7 @@ import { Login } from "./pages/Login/Login";
 import { SignUp } from "./pages/Signup/Signup";
 import { Playlist } from "./pages/Playlist/Playlist";
 import { SinglePlaylist } from "./pages/SinglePlaylist/SinglePlaylist";
+import { WatchLater } from "./pages/WatchLater/WatchLater";
```

Comment:
You can use barrel imports to avoid repetition

Reply:
thanks for the suggestion, will add barrel imports